pvlib/tests/test_singlediode.py

```diff
@@ -185,3 +185,103 @@ def test_pvsyst_recombination_loss(method, poa, temp_cell, expected, tol):
 
     vsc_88 = bishop88_v_from_i(isc_88, *x, **y, method=method)
     assert np.isclose(vsc_88, 0.0, *tol)
+
+
+@requires_scipy
+@pytest.mark.parametrize(
+    'brk_params, recomb_params, poa, temp_cell, expected, tol', [
+        # reference conditions without breakdown model
+        (
+            (0., -5.5, 3.28),
+            (get_pvsyst_fs_495()['d2mutau'],
+             VOLTAGE_BUILTIN * get_pvsyst_fs_495()['cells_in_series']),
+            get_pvsyst_fs_495()['irrad_ref'],
+            get_pvsyst_fs_495()['temp_ref'],
+            {
+                'pmp': (get_pvsyst_fs_495()['I_mp_ref'] *  # noqa: W504
+                        get_pvsyst_fs_495()['V_mp_ref']),
+                'isc': get_pvsyst_fs_495()['I_sc_ref'],
+                'voc': get_pvsyst_fs_495()['V_oc_ref']
+            },
+            (5e-4, 0.04)
+        ),
+        # other conditions with breakdown model on and recombination model off
+        (
+            (1.e-4, -5.5, 3.28),
+            (0., np.Inf),
+            POA,
+            TCELL,
+            {
+                'pmp': 79.723,
+                'isc': 1.4071,
+                'voc': 79.646
+            },
+            (1e-4, 1e-4)
+        )
+    ]
+)
+@pytest.mark.parametrize('method', ['newton', 'brentq'])
+def test_pvsyst_breakdown(method, brk_params, recomb_params, poa, temp_cell,
+                          expected, tol):
+    """test PVSst recombination loss"""
```

Comment:
```suggestion
    """test PVSyst recombination loss"""
```
Perhaps remove/change to align with test name.